Description: Add Ubuntu token to NGINX_VER
Author: Adam Conrad <adconrad@ubuntu.com>
Forwarded: not-needed
Last-Update: 2014-03-09

Index: b/src/core/nginx.h
===================================================================
--- a/src/core/nginx.h
+++ b/src/core/nginx.h
@@ -11,7 +11,7 @@
 
 #define nginx_version      1004006
 #define NGINX_VERSION      "1.4.6"
-#define NGINX_VER          "nginx/" NGINX_VERSION
+#define NGINX_VER          "nginx/" NGINX_VERSION " (Ubuntu)"
 
 #define NGINX_VAR          "NGINX"
 #define NGX_OLDPID_EXT     ".oldbin"
